packages/tldraw/src/lib/ui/components/primitives/Kbd.tsx

```diff
@@ -7,6 +8,8 @@ export interface KbdProps {
 
 /** @internal */
 export function Kbd({ children }: KbdProps) {
+	const breakpoint = useBreakpoint()
+	if (breakpoint < 4) return null
```

Comment:
Feels slightly weird putting this breakpoint guard here, but it's what our CSS is currently doing. I'm happy to keep the same behaviour, but I'd also understand if we'd prefer to move this behaviour up to a higher level.